internal/core/metadata/rest_addressable.go

```diff
@@ -362,11 +362,25 @@ func restGetAddressableByAddress(w http.ResponseWriter, r *http.Request) {
 
 // Notify the associated device services for the addressable
```

Comment:
@jduranf I think you'll want to update this comment to reflect your change. I see you've eliminated the notification to the Device Service itself per your comment in the issue. Just want to confirm we don't want to maintain the old behavior in addition to your change.

Once above is addressed and branch is rebased, will merge.